config/locales/en.yml

```diff
@@ -1,31 +1,19 @@
-# Files in the config/locales directory are used for internationalization and
-# are automatically loaded by Rails. If you want to use locales other than
-# English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t "hello"
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t("hello") %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# To learn more about the API, please read the Rails Internationalization guide
-# at https://guides.rubyonrails.org/i18n.html.
-#
-# Be aware that YAML interprets the following case-insensitive strings as
-# booleans: `true`, `false`, `on`, `off`, `yes`, `no`. Therefore, these strings
-# must be quoted to be interpreted as strings. For example:
-#
-#     en:
-#       "yes": yup
-#       enabled: "ON"
-
 en:
-  hello: "Hello world"
+  registration:
+    title: "Register"
+    submit_button: "Register"
+  terms_and_services:
+    title: "Terms and Services"
+    agreement_text: "I have read and understood the"
+    link_text: "Terms and Services"
+    accept_button: "I Understand"
+    content: |
+      1. Acceptance of Terms:
+      
+      By accessing or using our Service, you acknowledge that you have read, understood, and agree to be bound by these Terms.
+      
+      2. Changes to Terms:
+      We reserve the right to modify or update these Terms at any time.
```

Comment:
Bu şekilde yaparsak tek dosyamızda çok fazla uzunluğa gitmemiz gerekir bunun yerine amaca uygun bunları ayırmak daha rahat olacaktır. Örneğin view için;
config/locales/app/views/registrations.en.yml

gibi